snap/local/config/freedoom.wrapper

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# copy default settings if they don't currently exist
+if [[ ! -f "$SNAP_USER_DATA/.config/gzdoom/gzdoom.ini" ]]; then
+    mkdir $SNAP_USER_DATA/.config 2>/dev/null
+    mkdir $SNAP_USER_DATA/.config/gzdoom 2>/dev/null
```

Comment:
Combine these two lines by using `mkdir -p`, and don't redirect stderr.

Reply:
Can't do `mkdir -p` here, permissions issues with snap containment.

Reply:
Do you mind elaborating on this?

Also, I thought of something else: is `$SNAP_USER_DATA` the same as `$HOME` or is it some kind of special sandboxing environment? I'm thinking it is possible that ~/.config doesn't actually apply universally (since `$XDG_CONFIG_HOME` is what should be used...)

Reply:
Basically, because of the apparmor profile that's applied to the snap, attempting to run mkdir -p with an absolute path will fail to run because apparmor will not allow it to even stat certain directories in its path. Trust me, I've torn my hair out over this enough times :P 
 
`$SNAP_USER_DATA` is a special sandboxing environment, yes. It's where all the snap user data gets put instead of `$HOME` (because it doesn't have permission to write to hidden files in the home directory).